content/en/blog/_posts/2025-12-17-kubernetes-v1-35-release/index.md

```diff
@@ -0,0 +1,476 @@
+---  
+layout: blog
+title: "Kubernetes v1.35: TBD"
+date: 2025-12-17T10:30:00-08:00
+draft: true
+evergreen: true
+slug: kubernetes-v1-35-release
+author: >
+  [Kubernetes v1.35 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.35/release-team.md)
+---
+
+**Editors**: Aakanksha Bhende, Arujjwal Negi, Chad M. Crowell, Graziano Casto, Swathi Rao
+
+Similar to previous releases, the release of Kubernetes v1.35 introduces new stable, beta, and alpha features. The consistent delivery of high-quality releases underscores the strength of our development cycle and the vibrant support from our community.
+
+This release consists of 60 enhancements. Of those enhancements, 17 are Stable, 19 Beta, and 22 Alpha.
+
+There are also some [deprecations and removals](#deprecations-and-removals) in this release; make sure to read about those.
+
+## Release theme and logo
+
+{{< figure src="TBD" alt="Kubernetes v1.35 logo: TBD" class="release-logo" >}}
+
+TBD
```

Comment:
I believe this can be updated now that we have https://github.com/kubernetes/sig-release/tree/master/releases/release-1.35/logo